### PR TITLE
Fix: Correct way for popups checking if user is logged in

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/router/index.ts [hunk 1]
```diff
@@ -58,8 +58,14 @@ RouteQueryActionsController.init(router);
 RouteLastUsedRoutes.init(router);
 
 router.beforeEach(async (to, from, next) => {
+  // Move the route resolving to the next thread cycle in order to allow the app
+  // to access the stored data (localStorage/browserStorage) first.
+  await new Promise((resolve) => setTimeout(resolve, 0));
```
Comment (martinkaintas) [inline src/popup/router/index.ts:63]: does it make sense to use `nextTick()` from vue here?

**Review by martinkaintas — APPROVED** [review]

File: src/popup/router/index.ts (hunk 1 above)
Comment (CedrikNikita) [reply src/popup/router/index.ts:63]: This looks like a workaround to me, then an actual good decision.

File: src/composables/accounts.ts [hunk 2]
```diff
@@ -26,6 +26,8 @@ import { useStorageRef } from './storageRef';
 
 let isInitialized = false;
 
+const isAccountsRestored = ref(false);
```
Comment (peronczyk) [inline src/composables/accounts.ts:29]: Do you think this name is OK? Or maybe `areAccountsRestored`?

Comment (CedrikNikita) [reply src/composables/accounts.ts:29]: `areAccountsRestored` is better.

**Pull request merged**